app/models/chess_piece.rb

```diff
@@ -1,5 +1,8 @@
 # Chess piece model
 class ChessPiece < ActiveRecord::Base
+  BOARD_START_INDEX = 1
```

Comment:
Super nitpicky/crazy, but if you name this something like `BOARD_BGN_INDEX` or something else with 3 letters, the body of the method will align nicely. :smile_cat: